updated_OU.main.Rmd

```diff
@@ -2,9 +2,9 @@
 title: "A Hierarchical extension to Ornstein-Uhlenbeck-type Student's t-processes"
 author: |
   | Ville Laitinen & Leo Lahti 
-  | University of Turku
+  | University of Turku, Finland
   | velait@utu.fi
-date: "15.4.2018"
+date: "July 20, 2018"
```

Comment:
This could also be: 

`date: "`r Sys.Date()`"`

In that case R will automatically generate the up-to-date date information.

Reply:
ok

Reply:
Jeee!